lib/Doctrine/ORM/Query/ResultSetMappingBuilder.php

```diff
@@ -16,6 +17,7 @@
 use function sprintf;
 use function strpos;
 use function strtolower;
+use function var_dump;
```

Comment:
I believe this is unused, not sure why phpcs/psalm don't complain

Reply:
That would be because of this: https://github.com/doctrine/orm/blob/760397c429362114e18e8420a6eaab0a2d43aa1c/phpcs.xml.dist#L34-L35

Reply:
Ah forgot about having to add that myself :)

Reply:
Fixed in https://github.com/doctrine/orm/pull/9338